app/views/offers/show.html.erb

```diff
@@ -1,10 +1,17 @@
-<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(https://images.unsplash.com/photo-1562436260-8c9216eeb703?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1341&q=80);">
+<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url('<%= cl_image_path @offer.photo.key, crop: :fill %>')">
   <div class="container">
     <h1><%= @offer.title %></h1>
     <p><%= @offer.description %></p>
     <a class="btn btn-flat" href="#">Book now</a>
-  <span class='person-tag'><%= pluralize(@offer.number_of_people.size, 'person') %></span>
+  <span class='person-tag'><%= pluralize(@offer.number_of_people, 'person') %></span>
   <span class='category-tag'><%= @offer.category %></span>
-  <span class='price-tag'><%= @offer.price %>EUR/pers</span>
+  <span class='price-tag'><%= @offer.price.round %>EUR/pers</span>
   </div>
 </div>
+
+<%= link_to "Back to offers", offers_path %>
+<%= link_to "Edit this offer", edit_offer_path %>
```

Comment:
For Edit and Delete we'll need to check if the current logged in user is the one who created this offer.

Reply:
No, I meant without Pundit, just with conditional statements. 
I think we didn't decide if we wanted to implement Pundit or not.
If we do decide to add Pundit, i think we should merge this code first and then start Pundit integration on a new branch.